cylc/flow/parsec/validate.py

```diff
@@ -699,6 +700,30 @@ class CylcConfigValidator(ParsecValidator):
                 '-0830': 'UTC minus 8 hours and 30 minutes.'
             }
         ),
+        V_CYCLE_POINT_WITH_OFFSETS: (
```

Comment:
This is a new parsec validator type which does the same thing as `V_STRING`.

It exists purely for documentation purposes.